src/osu_one_sided.jl

```diff
@@ -0,0 +1,65 @@
+function run_osu_one_sided(benchmark::MPIBenchmark, func::Function, conf::Configuration)
+    if (conf.synchronization_option !== "lock" && conf.synchronization_option !== "fence")
```

Comment:
This is a bit simpler:
```suggestion
    if conf.synchronization_option ∉ ("lock", "fence")
```